skimage/feature/blob.py

```diff
@@ -429,8 +451,8 @@ def blob_log(image, min_sigma=1, max_sigma=50, num_sigma=10, threshold=.2,
 
     # computing gaussian laplace
     # average s**2 provides scale invariance
-    gl_images = [-gaussian_laplace(image, s) * s ** 2
-                 for s in np.mean(sigma_list, axis=1)]
+    gl_images = [-gaussian_laplace(image, s) * np.mean(s) ** 2
+                 for s in sigma_list]
```

Comment:
Did you intentionally not apply `np.mean` to the `s` given to `gaussian_laplace`? Or should this rather be `gaussian_laplace(image, np.mean(s))`?

Reply:
Yes, this was a bug in the original implementation, see `blob_dog` for the equivalent code. It makes no sense to have this accept multiple sigma if you're just going to take the mean of them!

Reply:
oh yes that was a bug! For the scaling factor, is taking the `mean` rather than another statistical value (median, harmonic mean, etc.) arbitrary? I guess it's not very important.